components/model.py

```diff
@@ -0,0 +1,83 @@
+########
+## model.py
+## Handles queries to the database
+########
+
+import web
+import psycopg2
+import components.database_adapter # database_adaptor.py handles the connection to database
+
+## Connects to the postgres database
+connection = components.database_adapter.connect_db()
+db_cursor = connection.cursor()
+
+## Get the module code, name, description, and MCs of all modules
+def getAllModules():
+	sql_command = "SELECT * FROM module ORDER BY code"
+	db_cursor.execute(sql_command)
+	return db_cursor.fetchall()
+
```

Comment:
Team-agreed convention to use 2 newlines after each function definition, to add 1 more newline.